extensions/random-password-generator/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "random-password-generator",
+  "title": "Random Password Generator",
+  "description": "Generate random password of different length",
+  "icon": "icon.png",
+  "author": "textnav",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Generate random password",
```

Comment:
```suggestion
      "title": "Generate Random Password",
```

Reply:
@fe9lix let's commit this kind of change into the PR straight away.